plutus-core/plutus-ir/test/PlutusIR/Transform/Inline/letNonPureMultiStrict.golden

```diff
@@ -1,11 +1,4 @@
 (let
-  (nonrec)
-  (termbind
-    (strict)
-    (vardecl y (con integer))
-    [
-      [ { (builtin trace) (con integer) } (con string "hello") ] (con integer 1)
-    ]
-  )
-  [ [ (builtin addInteger) y ] [ [ (builtin addInteger) y ] y ] ]
-)
\ No newline at end of file
+  !y-3 : integer = trace {integer} "hello" 1
```

Comment:
I don't think we need uniques in all these gold files. IMO there are only a handful of tests whose golden files need uniques for disambiguation.